python/lsst/ip/isr/isrFunctions.py

```diff
@@ -484,7 +484,7 @@ def overscanCorrection(ampMaskedImage, overscanImage, fitType='MEDIAN', order=1,
             (`lsst.afw.image.Image`)
     Raises
     ------
-    pexExcept.Exception
+    RuntimeError
```

Comment:
I'd suggest `ValueError` for that, but either way is clearly an improvement :)

Reply:
Well this is documentation so the question is what happens when a config item is assigned a value that is not allowed. Is it a RuntimeError or a ValueError? It's not open to debate since the code is the code. It's clearly not a pex exception anyhow.

Reply:
They don't have a freeform emoji response here, but if they did, I'd have used 🔥 , fwiw.